src/main/java/org/opentripplanner/middleware/utils/ApiGatewayUtils.java

```diff
@@ -136,16 +162,30 @@ public static void deleteApiKeys(List<String> apiKeyIds) {
                 usageResults.add(gateway.getUsage(getUsageRequest));
             } catch (Exception e) {
                 // Catch any issues with bad request parameters (e.g., invalid API keyId or bad date format).
-                LOG.error("Unable to get usage results for key id ({}) with start date ({}) and end date ({})",
+                String message = String.format("Unable to get usage results for key id (%s) with start date (%s) and end date (%s)",
```

Comment:
Suggestion: "Unable to get usage results for key id (%s) between (%s) and (%s)"

Reply:
Updated.